### PR TITLE
move header and footer to component folders

### PR DESCRIPTION
**Pull request by dcwebermn**

### CODE REVIEW

**Review by cagrev — COMMENTED** [review]
Looks ok to me, but I'm not able to load localhost:8000 after using gatsby develop, so there must still be something wrong in the config/setup.

**Comment by dcwebermn** [comment]
@cagrev Yes, the npm / gatsby dependency chain is a real mess with v4.x.  I got things building locally again and fixed the header / footer issue.

File: .github/workflows/acd-containers-build.yml [hunk 1]
```diff
@@ -28,15 +28,11 @@ jobs:
...
-      # Runs a single command using the runners shell
-      - name: Run a one-line script
-        run: echo Hello, world!
-
       # Runs a set of commands using the runners shell
       - name: Run a multi-line script
         run: |
           npm install -g markdownlint-cli
-          npm ci --force
+          npm install -force
```
Comment (cagrev) [inline .github/workflows/acd-containers-build.yml:35]: Not sure I fully understand the difference between the clean install and regular install, but appears to be working again and the pages load. Thanks.

**Review by cagrev — APPROVED** [review]
I still see Carbon Design System in the upper left corner when you have time to address that.

File: .github/workflows/acd-containers-build.yml (hunk 1 above)
Comment (dcwebermn) [reply .github/workflows/acd-containers-build.yml:35]: Clean install does deeper analysis of chained dependencies which eventually we want to get back to but there is a lot of compatibility issues between Gatsby and Gatsby plugins that need to be worked through.  Normally we keep the dependency list in package.json as short as possible to help with this but a lot has been included and they have conflicting version declarations.  I'll work through that in the background as it doesn't impact the gh-pages deployment of the documentation.

**Pull request merged**